schema_forms/mammo_form.py

```diff
@@ -36,75 +38,77 @@ class MammoMassForm(FlaskForm):
     class Meta:
         csrf = False
 
-    mammo_mass_number = IntegerField("Mass detected", default=1)
-    mammo_mass_location_right_breast = SelectField("Location of mass in Right Breast",
+    fld_number = IntegerField("Mass detected",[validators.optional()])
```

Comment:
>[validators.optional()] [](start = 46, length = 23)

from: [https://stackoverflow.com/questions/21451058/integerfield-and-required-validator-unintuitive-always-required](https://stackoverflow.com/questions/21451058/integerfield-and-required-validator-unintuitive-always-required)